python/create-croissant-client-side/requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/Dans-labs/pyDataverse@development#egg=pyDataverse 
```

Comment:
Hmm, what about all the extra stuff? --upgrade --no-cache-dir  --break-system-packages

Here was the original command:

`pip install --upgrade --no-cache-dir  git+https://github.com/Dans-labs/pyDataverse@development#egg=pyDataverse --break-system-packages`

Reply:
Hmm, I think this is impossible because the extra flags are catered to the `pip install` command rather than the URL or package name.

Reply:
Ok, I'll try it. Maybe we don't need those flags. 🤷 Thanks!

Reply:
* `--upgrade` is only relevant if pyDataverse is already installed. I would leave that one out.
* `--no-cache-dir` is not necessary most of the time. 
* `--break-system-packages`is only relevant for Linux systems *if* any system packages are installed, but to my knowledge there is none for pyDataverse